docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -157,6 +155,36 @@ processed after a dynamic filter is pushed down to the table scan.
     "inputPositions" : 28800991,
     "dynamicFilterSplitsProcessed" : 1,
 
+Dynamic filters collected by the coordinator are also reported as a part of the
+:doc:`EXPLAIN ANALYZE plan </sql/explain-analyze>` in the statistics for
+``ScanFilterProject`` nodes.
+
+.. code-block:: text
+
+    ...
+
+     └─ InnerJoin[("ss_sold_date_sk" = "d_date_sk")][$hashvalue, $hashvalue_4]
+        │   Layout: []
+        │   Estimates: {rows: 11859 (0B), cpu: 8.84M, memory: 3.19kB, network: 3.19kB}
+        │   CPU: 78.00ms (30.00%), Scheduled: 295.00ms (47.05%), Output: 296 rows (0B)
+        │   Left (probe) Input avg.: 120527.00 rows, Input std.dev.: 0.00%
+        │   Right (build) Input avg.: 0.19 rows, Input std.dev.: 208.17%
+        │   Collisions avg.: 0.00 (0.00% est.), Collisions std.dev.: ?%
+        │   Distribution: REPLICATED
+        │   dynamicFilterAssignments = {d_date_sk -> #df_370}
+        ├─ ScanFilterProject[table = hive:default:store_sales, grouped = false, filterPredicate = true, dynamicFilters = {"ss_sold_date_sk" = #df_370}]
+        │      Layout: [ss_sold_date_sk:bigint, $hashvalue:bigint]
+        │      Estimates: {rows: 120527 (2.03MB), cpu: 1017.64k, memory: 0B, network: 0B}/{rows: 120527 (2.03MB), cpu: 1.99M, memory: 0B, network: 0B}/{rows: 120527 (2.03MB), cpu: 4.02M, memory: 0B, network: 0B}
+        │      CPU: 49.00ms (18.85%), Scheduled: 123.00ms (19.62%), Output: 120527 rows (2.07MB)
+        │      Input avg.: 120527.00 rows, Input std.dev.: 0.00%
+        │      $hashvalue := combine_hash(bigint '0', COALESCE("$operator$hash_code"("ss_sold_date_sk"), 0))
+        │      ss_sold_date_sk := ss_sold_date_sk:bigint:REGULAR
+        │      Input: 120527 rows (1.03MB), Filtered: 0.00%
+        │      Dynamic filters:
+        │          - df_370, [ SortedRangeSet[type=bigint, ranges=3, {[2451546], ..., [2451905]}] ], collection time=2.34s
+    
```

Comment:
you could add remaining `ScanFilterProject` lines too

Reply:
That is the end of output for ScanFilterProject, subsequent lines are for other parts of the plan